backend/src/akvo/lumen/auth.clj

```diff
@@ -57,17 +59,18 @@
 (defn wrap-jwt
   "Go get cert from Keycloak and feed it to wrap-jwt-claims. Keycloak url can
   be configured via the KEYCLOAK_URL env var."
-  [{:keys [url realm]}]
+  [keycloak]
   (fn [handler]
-   (try
-     (let [issuer (str url "/realms/" realm)
-           certs  (-> (str issuer "/protocol/openid-connect/certs")
-                      client/get
-                      :body)]
-       (jwt/wrap-jwt-claims handler (jwt/rsa-key certs 0) issuer))
-     (catch Exception e
-       (println "Could not get cert from Keycloak")
-       (throw e)))))
+    (let [keycloak-verifier (RSASSAVerifier. (:rsa-key keycloak))]
```

Comment:
This reminds me that in theory, for additional security, the RSA key from the OAuth server should rotate periodically. Not sure why we did not enable that feature in Keycloak, but the code as it is/was will not support the RSA key rotation.

It is ok for this pull request but please keep it in mind now that you are our security expert.

Reply:
I'll keep it in mind 👍 !